Dockerfile

```
@@ -0,0 +1,79 @@
+# Install dependencies only when needed
+FROM node:14 AS deps
+
+ARG TWOOPSTRACKER_API_URL
+ARG NEXT_PUBLIC_TWOOPSTRACKER_API_URL
+ARG NEXTAUTH_PROVIDERS_OAUTH_LOGIN_URL
+ARG NEXTAUTH_JWT_TOKEN_VERIFY_URL
+ARG NEXTAUTH_JWT_TOKEN_REFRESH_URL
+ARG NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_SECRET
+ARG NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_ID
+ARG NEXTAUTH_URL
+ARG NEXT_PUBLIC_IMAGE_DOMAINS
+ARG NEXTAUTH_SECRET
+ARG S3_UPLOAD_SECRET
+ARG S3_UPLOAD_KEY
+ARG S3_UPLOAD_BUCKET
+ARG S3_UPLOAD_REGION
+ARG ADMIN_BACKEND_REPO
+ARG ADMIN_BACKEND_AUTH_ENDPOINT
+ARG ADMIN_OAUTH_CLIENT_ID
+ARG ADMIN_OAUTH_CLIENT_SECRET
+ARG ADMIN_OAUTH_SCOPE
+
+ENV TWOOPSTRACKER_API_URL=${TWOOPSTRACKER_API_URL} \
+    NEXT_PUBLIC_TWOOPSTRACKER_API_URL=${NEXT_PUBLIC_TWOOPSTRACKER_API_URL} \
+    NEXTAUTH_PROVIDERS_OAUTH_LOGIN_URL=${NEXTAUTH_PROVIDERS_OAUTH_LOGIN_URL} \
+    NEXTAUTH_JWT_TOKEN_VERIFY_URL=${NEXTAUTH_JWT_TOKEN_VERIFY_URL} \
+    NEXTAUTH_JWT_TOKEN_REFRESH_URL=${NEXTAUTH_JWT_TOKEN_REFRESH_URL} \
+    NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_SECRET=${NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_SECRET} \
+    NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_ID=${NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_ID} \
+    NEXTAUTH_URL=${NEXTAUTH_URL} \
+    NEXT_PUBLIC_IMAGE_DOMAINS=${NEXT_PUBLIC_IMAGE_DOMAINS} \
+    NEXTAUTH_SECRET=${NEXTAUTH_SECRET} \
+    S3_UPLOAD_SECRET=${S3_UPLOAD_SECRET} \
+    S3_UPLOAD_KEY=${S3_UPLOAD_KEY} \
+    S3_UPLOAD_BUCKET=${S3_UPLOAD_BUCKET} \
+    S3_UPLOAD_REGION=${S3_UPLOAD_REGION} \
+    ADMIN_BACKEND_REPO=${ADMIN_BACKEND_REPO} \
+    ADMIN_BACKEND_AUTH_ENDPOINT=${ADMIN_BACKEND_AUTH_ENDPOINT} \
+    ADMIN_OAUTH_CLIENT_ID=${ADMIN_OAUTH_CLIENT_ID} \
+    ADMIN_OAUTH_CLIENT_SECRET=${ADMIN_OAUTH_CLIENT_SECRET} \
+    ADMIN_OAUTH_SCOPE=${ADMIN_OAUTH_SCOPE} 
+
+WORKDIR /app
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
+
+# Rebuild the source code only when needed
+FROM node:14 AS builder
+
+WORKDIR /app
+
+COPY . .
+COPY --from=deps /app/node_modules ./node_modules
+RUN yarn build
+
+# Production image, copy all the files and run next
+FROM node:14 AS runner
+
+ENV NODE_ENV production
+
+WORKDIR /app
+
+RUN addgroup -g 1001 -S nodejs
+RUN adduser -S nextjs -u 1001
+
+
+# You only need to copy next.config.js if you are NOT using the default configuration
+# COPY --from=builder /app/next.config.js ./
```

```suggestion
COPY --from=builder /app/next.config.js ./
```
I think we're using custom `next.config.js`